x-pack/plugins/siem/public/components/page/network/domains_table/columns.tsx

```diff
@@ -146,40 +149,24 @@ export const getDomainsColumns = (
     },
   },
   {
-    name: i18n.FIRST_SEEN,
+    name: (
+      <EuiToolTip content={i18n.FIRST_LAST_SEEN_TOOLTIP}>
+        <span>
+          {i18n.LAST_SEEN}
+          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+        </span>
+      </EuiToolTip>
+    ),
```

Comment:
I tried this out and it looks like we can get away without using the `span` which would reduce some DOM weight for us.

```ts
 name: (
      <EuiToolTip content={i18n.FIRST_LAST_SEEN_TOOLTIP}>
        <>
          {i18n.LAST_SEEN}
          <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
        </>
      </EuiToolTip>
    ),
```